cluster/saltbase/salt/mount-propagation/init.sls

```diff
@@ -0,0 +1,15 @@
+{% if not pillar.get('is_systemd') %}
```

Comment:
I would leave it to non-systemd distros. Debian wheezy is the only one I tested, however there may be others that would need similar ~~hack~~ service.